packages/repository/test/unit/repositories/kv.repository.bridge.unit.ts

```diff
@@ -0,0 +1,105 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+
+import {
+  juggler,
+  DefaultKVRepository,
+  DataObject,
+  Model,
+  KVRepository,
+} from '../../../';
+
+describe('DefaultKVRepository', () => {
+  let ds: juggler.DataSource;
+  let kvNoteModel: typeof juggler.KeyValueModel;
+  let repo: KVRepository<Note>;
+
+  class Note extends Model {
+    title?: string;
+    content?: string;
+    id: number;
+
+    constructor(data: DataObject<Note>) {
+      super(data);
+    }
+  }
+
+  beforeEach(() => {
+    ds = new juggler.DataSource({
+      name: 'db',
+      connector: 'kv-memory',
+    });
+    kvNoteModel = ds.createModel<typeof juggler.KeyValueModel>('note');
```

Comment:
IMO, users should not need to understand these low-level implementation details of the legacy juggler bridge. Instead, they should be able to use the same API as `DefaultCrudRepository` offers now.

```ts
repo = new DefaultKVRepository<Note>(Note, ds);
```